tests/unit/IonBinaryWriterTest.js

```diff
@@ -854,6 +854,38 @@ define([
       assert.deepEqual(bytes, new Uint8Array([0xe3, 0x81, 0x8a, 0xbf]));
     }
 
+    //depth tests
+      var depthTest = function(name, instructions, expectedDepth) {
+          suite[name] = function() {
+              var symbolTable = new ion.LocalSymbolTable(ion.getSystemSymbolTableImport());
+              var writeable = new ion.Writeable();
+              var writer = new ion.BinaryWriter(symbolTable, writeable);
+              instructions(writer);
+              assert.equal(writer.depth(), expectedDepth);
+          }
+      }
+      depthTest('Stepping into a list results in a depth of 1.',
+          (writer) => {writer.stepIn(ion.IonTypes.LIST);}, 1);
```

Comment:
Did I miss a test for the zero case (top-level)?

Reply:
no I has the dumb